src/Components/Welcome.css

```diff
@@ -7,7 +7,17 @@ body {
   height: 100vh;
 }
 
-h2 {
+h4{
```

Comment:
These comments apply to the whole file:
- Don't use global CSS selectors to apply styles on a page.
- Always use classes and import them via PostCSS Modules to avoid global class names collision.
- Try not to use element selectors such as `input`, `h2`, etc. Classes are better to avoid style leaks outside of your components.
- Don't separate files into "sections" with comments (such as `/* STRUCTURE */`). If you need to separate a file into sections, then it's always better to divide your file into multiple files with descriptive names.